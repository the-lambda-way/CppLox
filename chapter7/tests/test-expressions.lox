2 * (3 / -"muffin")
