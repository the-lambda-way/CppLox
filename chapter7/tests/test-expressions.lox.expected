Operand must be a number.
[line 1]
